test/unit-test/catch_assert.h

```diff
@@ -62,17 +62,18 @@ static void catchHandler_( int signal )
         close( 2 );                          \
         if( setjmp( CATCH_JMPBUF ) == 0 )    \
         {                                    \
-            try++;                           \
+            try ++;                          \
```

Comment:
What does the space do?

Reply:
This is auto-formatted by the VS code extension I use for uncrustify.
Fixed